library/ssl_tls.c

```diff
@@ -379,7 +379,7 @@ static int tls_prf_generic( mbedtls_md_type_t md_type,
 {
     size_t nb;
     size_t i, j, k, md_len;
-    unsigned char tmp[128];
+    unsigned char tmp[144];
```

Comment:
check why changed to 144?

Reply:
without the change, the function fails because of:
~~~
    if( sizeof( tmp ) < md_len + strlen( label ) + rlen )
        return( MBEDTLS_ERR_SSL_BAD_INPUT_DATA );
~~~
`md_len` is 48 for SHA384 (when `MBEDTLS_SHA512_C` defined), `strlen("EXTRACTOR-dtls_srtp")` is 19 anmd `rlen` is 64. This sums to 131.
I'll change to 132, for now